docs/training.md

```diff
@@ -9,6 +9,39 @@ layout: default
 For easily training popular model types on known data sets, see the [Sockeye Autopilot documentation](https://github.com/awslabs/sockeye/tree/master/sockeye_contrib/autopilot).
 For manually training and running translation models on your data, read on.
 
+## Data preparation
+
+Sockeye can read the raw data at training time in two sentence-parallel files via the `--source` and `--target` command-line options.
+You can also prepare the data ahead of time and dump it to disk as MXNet NDArrays.
+This basically eliminates the data loading time when running training (since three passes over the raw data are required), and also reduces memory consumption, since prepared data is also placed into random shards (which have one million lines each, by default).
```

Comment:
I think its 10 million lines each per shard by default :)

Reply:
[Looks like 1m](https://github.com/awslabs/sockeye/blob/5a50d960b6e20527a006b176419b188d79a85a85/sockeye/arguments.py#L438). This value should probably be pulled out as a constant.

Reply:
you are right :)